spec.md

```diff
@@ -388,26 +388,47 @@ both appear within the same JSON object.
   or application protocol's content-type metadata property. Normative rules
   for the binary mode and the content-type metadata mapping can be found 
   in the respective transport mapping specifications.
-    
+  
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
-### extensions
+### identificationLabel
 * Type: `Map`
-* Description: This is for additional metadata and this does not have a
-  mandated structure. This enables a place for custom fields a producer or
-  middleware might want to include and provides a place to test metadata before
-  adding them to the CloudEvents specification.
-  See the [Extensions](extensions.md) document for a list of possible
-  attributes.
+* Description: A list of "key-value" pairs providing identification
```

Comment:
I highly recommend a list of tuples rather than a map. Simple example:


This PR adds:

```
"identificationLabel":{
   "employee-ID": "H0098001",
   "traveler-name": "Harry Fisher",
   "travel-request-ID": "123456"
}
```

But I would recommend the following:

"labels":[
  {
   "name": "employee-ID",
    "value": "H0098001",
  },{
   "name": "traveler-name",
    "value": "Harry Fisher",
  },{ 
   "name":"travel-request-ID",
    "value":"123456",
  }
]
```

And the benefit is that you can add other meanings to the tuple without breaking the API or adding something that needs to remap, example:

```
"labels":[
  {
   "name": "employee-ID",
    "value": "H0098001",
    "type": "id"
  },{
   "name": "traveler-name",
    "value": "Harry Fisher",
    "type": "name"
  },{ 
   "name":"travel-request-ID",
    "value":"123456",
    "type": "int"
  }
]
```

Now the processor of the event can look at the meaning of the labels and understand how to process it. If you need to you can flatten it into a map on the processing side and then get O(1) lookups.